src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -178,6 +178,7 @@ Copyright (C) 2011-2012 Xamarin. All rights reserved.
 	<MonoAndroidVersion>v$(_XAMajorVersionNumber).0</MonoAndroidVersion>
 	<AndroidUpdateResourceReferences Condition="'$(AndroidUpdateResourceReferences)' == ''">True</AndroidUpdateResourceReferences>
 	<EmbedAssembliesIntoApk Condition=" '$(EmbedAssembliesIntoApk)' == '' Or '$(_XASupportsFastDev)' == 'False' ">True</EmbedAssembliesIntoApk>
+	<UseRuntimeWithDebugSymbols Condition=" '$(UseRuntimeWithDebugSymbols)' == '' ">False</UseRuntimeWithDebugSymbols>
```

Comment:
This should have an `Android` prefix, and should be similarly renamed as per a prior comment.